hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/JobCoordinationService.java

```diff
@@ -998,4 +1016,39 @@ NodeEngineImpl nodeEngine() {
     void assertOnCoordinatorThread() {
         assert IS_JOB_COORDINATOR_THREAD.get() : "not on coordinator thread";
     }
+
+    private static class Stats {
+
+        @Probe(name = "jobs.submitted")
+        private final AtomicInteger jobSubmitted = new AtomicInteger();
+        @Probe(name = "jobs.completed_successfully")
+        private final AtomicInteger jobCompletedSuccessfully = new AtomicInteger();
+        @Probe(name = "jobs.completed_with_failure")
```

Comment:
looks like we don't use consistent naming with metrics. See `MetricNames`

Reply:
Fixed.